src/vs/workbench/contrib/debug/browser/rawDebugSession.ts

```diff
@@ -713,9 +713,12 @@ export class RawDebugSession implements IDisposable {
 		const url = error?.url;
 		if (error && url) {
 			const label = error.urlLabel ? error.urlLabel : nls.localize('moreInfo', "More Info");
+			const uri = URI.parse(url);
+			// Use a suffixed id if uri invokes a command, so default 'Open launch.json' command is suppressed on dialog
+			const actionId = uri.scheme === 'command' ? 'debug.moreInfo.command' : 'debug.moreInfo';
```

Comment:
Please use this `scheme const` for `command`
https://github.com/microsoft/vscode/blob/main/src/vs/base/common/network.ts#L49

Reply:
Done